server/src-lib/Hasura/Server/Auth/JWT.hs

```diff
@@ -155,35 +152,22 @@ updateJwkRef (Logger logger) manager url jwkRef = do
     writeIORef jwkRef jwkset
 
   -- first check for Cache-Control header to get max-age, if not found, look for Expires header
-  let cacheHeader   = resp ^? Wreq.responseHeader "Cache-Control"
-      expiresHeader = resp ^? Wreq.responseHeader "Expires"
-  case cacheHeader of
-    Just header -> getTimeFromCacheControlHeader header
-    Nothing     -> mapM getTimeFromExpiresHeader expiresHeader
+  let cacheHeader   = bsToTxt <$> resp ^? Wreq.responseHeader "Cache-Control"
+      expiresHeader = bsToTxt <$> resp ^? Wreq.responseHeader "Expires"
+  cct <- timeFromCacheControlHeader cacheHeader $ logAndThrowInfo . parseCacheControlErr
```

Comment:
Please don’t use variable names like `cct` if you can help it! Our [coding style](https://github.com/hasura/graphql-engine/blob/950d3842e4662c2ff5d8d5b20ac64bf9f9387b80/server/CONTRIBUTING.md#naming) is to avoid abbreviations. A name like `cacheDuration` isn’t too long, I think, and it’s much easier to read!

Reply:
My bad!